Update wording + add `,` as key for major mode (#9)
VSpaceCode/vspacecode.github.io: docs/major-mode.md

title: Major Mode
---

> Major mode offers dynamic menu and functions based on the activate buffer. You can access key bindings specific to the current major mode by pressing `<spc> m`. In addition, `,` is bound to major mode menu to save one keystroke.

Reviewer: ```suggestion
Major mode offers dynamic menu and functions based on the active buffer. You can access key bindings specific to the current major mode by pressing `<spc> m` or `,` if you want to save one keystroke.
```

For example, you can press `<spc> m x b` to make the text bold in a markdown buffer.

> To find the activate language mode of your current buffer, you can look the bottom right on the status bar.

Reviewer: ```suggestion
To find the active language mode of your current buffer, you can look the bottom right of the status bar.
```

![major mode position](/img/docs/major_mode.png)

> The vast majority of major mode specific key bindings will require the installation of additional extensions in order to work properly. The section below lists all the extensions required by each language in the major mode.

Reviewer: ```suggestion
The vast majority of major mode specific key bindings will require the installation of additional extensions in order to work properly. The section below lists all the extensions required by each major mode.
```
